controllers/admin/AdminThemesController.php

```diff
@@ -438,6 +439,7 @@ public function renderOptions()
                     ),
                     'PS_FAVICON' => array(
                         'title' => $this->trans('Favicon', array(), 'Admin.Design.Feature'),
+                        'desc' => $this->trans('Use our %url% to boost your brand image!', array('%url%' => $url), 'Admin.Design.Help'),
```

Comment:
Usually we create only one string to be translated, to make the context easier to understand.
URL tags are replaced by `[1]` and `[/1]` placeholders.

An example can be found here: https://github.com/PrestaShop/PrestaShop/blob/4a3b9894c083315768cebe912aaad355c49657ba/controllers/admin/AdminLoginController.php#L78-L82

Reply:
Done @Quetzacoalt91 :+1: